Build on windows with python3.6 fails

Building fabio on Windows for Python 3.6-64 bits failed:

```
C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\BIN\x86_amd64\cl.exe /c /
nologo /Ox /W3 /GL /DNDEBUG /MD -Ifabio\ext\include -Ifabio\ext -IC:\Python36\li
b\site-packages\numpy\core\include -IC:\Python36\include -IC:\Python36\include -
I"C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\INCLUDE" -I"C:\Program
Files (x86)\Microsoft Visual Studio 14.0\VC\ATLMFC\INCLUDE" -I"C:\Program Files
(x86)\Windows Kits\10\include\10.0.10586.0\ucrt" -I"C:\Program Files (x86)\Windo
ws Kits\NETFXSDK\4.6.1\include\um" -I"C:\Program Files (x86)\Windows Kits\10\inc
lude\10.0.10586.0\shared" -I"C:\Program Files (x86)\Windows Kits\10\include\10.0
.10586.0\um" -I"C:\Program Files (x86)\Windows Kits\10\include\10.0.10586.0\winr
t" /Tcfabio\ext\mar345_IO.c /Fobuild\temp.win-amd64-3.6\Release\fabio\ext\mar345
_IO.obj
mar345_IO.c
c:\python36\lib\site-packages\numpy\core\include\numpy\npy_1_7_deprecated_api.h(
12) : Warning Msg: Using deprecated NumPy API, disable it by #defining NPY_NO_DE
PRECATED_API NPY_1_7_API_VERSION
c:\users\tvincent\desktop\fabio\fabio\ext\include\msvc\\stdint.h(105): error C23
71: 'int_fast16_t': redefinition; different basic types
C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\INCLUDE\stdint.h(36): not
e: see declaration of 'int_fast16_t'
c:\users\tvincent\desktop\fabio\fabio\ext\include\msvc\\stdint.h(109): error C23
71: 'uint_fast16_t': redefinition; different basic types
C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\INCLUDE\stdint.h(40): not
e: see declaration of 'uint_fast16_t'
c:\users\tvincent\desktop\fabio\fabio\ext\include\msvc\\stdint.h(132): warning C
4005: 'INT8_MIN': macro redefinition
C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\INCLUDE\stdint.h(48): not
e: see previous definition of 'INT8_MIN'
c:\users\tvincent\desktop\fabio\fabio\ext\include\msvc\\stdint.h(133): warning C
4005: 'INT8_MAX': macro redefinition
C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\INCLUDE\stdint.h(52): not
e: see previous definition of 'INT8_MAX'
c:\users\tvincent\desktop\fabio\fabio\ext\include\msvc\\stdint.h(134): warning C
4005: 'INT16_MIN': macro redefinition
C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\INCLUDE\stdint.h(49): not
e: see previous definition of 'INT16_MIN'
c:\users\tvincent\desktop\fabio\fabio\ext\include\msvc\\stdint.h(135): warning C
4005: 'INT16_MAX': macro redefinition
C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\INCLUDE\stdint.h(53): not
e: see previous definition of 'INT16_MAX'
c:\users\tvincent\desktop\fabio\fabio\ext\include\msvc\\stdint.h(136): warning C
4005: 'INT32_MIN': macro redefinition
C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\INCLUDE\stdint.h(50): not
e: see previous definition of 'INT32_MIN'
c:\users\tvincent\desktop\fabio\fabio\ext\include\msvc\\stdint.h(137): warning C
4005: 'INT32_MAX': macro redefinition
C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\INCLUDE\stdint.h(54): not
e: see previous definition of 'INT32_MAX'
c:\users\tvincent\desktop\fabio\fabio\ext\include\msvc\\stdint.h(138): warning C
4005: 'INT64_MIN': macro redefinition
C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\INCLUDE\stdint.h(51): not
e: see previous definition of 'INT64_MIN'
c:\users\tvincent\desktop\fabio\fabio\ext\include\msvc\\stdint.h(139): warning C
4005: 'INT64_MAX': macro redefinition
C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\INCLUDE\stdint.h(55): not
e: see previous definition of 'INT64_MAX'
c:\users\tvincent\desktop\fabio\fabio\ext\include\msvc\\stdint.h(140): warning C
4005: 'UINT8_MAX': macro redefinition
C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\INCLUDE\stdint.h(56): not
e: see previous definition of 'UINT8_MAX'
c:\users\tvincent\desktop\fabio\fabio\ext\include\msvc\\stdint.h(141): warning C
4005: 'UINT16_MAX': macro redefinition
C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\INCLUDE\stdint.h(57): not
e: see previous definition of 'UINT16_MAX'
c:\users\tvincent\desktop\fabio\fabio\ext\include\msvc\\stdint.h(142): warning C
4005: 'UINT32_MAX': macro redefinition
C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\INCLUDE\stdint.h(58): not
e: see previous definition of 'UINT32_MAX'
c:\users\tvincent\desktop\fabio\fabio\ext\include\msvc\\stdint.h(143): warning C
4005: 'UINT64_MAX': macro redefinition
C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\INCLUDE\stdint.h(59): not
e: see previous definition of 'UINT64_MAX'
c:\users\tvincent\desktop\fabio\fabio\ext\include\msvc\\stdint.h(162): warning C
4005: 'INT_FAST16_MIN': macro redefinition
C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\INCLUDE\stdint.h(75): not
e: see previous definition of 'INT_FAST16_MIN'
c:\users\tvincent\desktop\fabio\fabio\ext\include\msvc\\stdint.h(163): warning C
4005: 'INT_FAST16_MAX': macro redefinition
C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\INCLUDE\stdint.h(79): not
e: see previous definition of 'INT_FAST16_MAX'
c:\users\tvincent\desktop\fabio\fabio\ext\include\msvc\\stdint.h(169): warning C
4005: 'UINT_FAST16_MAX': macro redefinition
C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\INCLUDE\stdint.h(83): not
e: see previous definition of 'UINT_FAST16_MAX'
c:\users\tvincent\desktop\fabio\fabio\ext\include\msvc\\stdint.h(192): warning C
4005: 'PTRDIFF_MIN': macro redefinition
C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\INCLUDE\stdint.h(101): no
te: see previous definition of 'PTRDIFF_MIN'
c:\users\tvincent\desktop\fabio\fabio\ext\include\msvc\\stdint.h(193): warning C
4005: 'PTRDIFF_MAX': macro redefinition
C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\INCLUDE\stdint.h(102): no
te: see previous definition of 'PTRDIFF_MAX'
c:\users\tvincent\desktop\fabio\fabio\ext\include\msvc\\stdint.h(199): warning C
4005: 'SIG_ATOMIC_MIN': macro redefinition
C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\INCLUDE\stdint.h(108): no
te: see previous definition of 'SIG_ATOMIC_MIN'
c:\users\tvincent\desktop\fabio\fabio\ext\include\msvc\\stdint.h(200): warning C
4005: 'SIG_ATOMIC_MAX': macro redefinition
C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\INCLUDE\stdint.h(109): no
te: see previous definition of 'SIG_ATOMIC_MAX'
c:\users\tvincent\desktop\fabio\fabio\ext\include\msvc\\stdint.h(218): warning C
4005: 'WINT_MIN': macro redefinition
C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\INCLUDE\stdint.h(114): no
te: see previous definition of 'WINT_MIN'
c:\users\tvincent\desktop\fabio\fabio\ext\include\msvc\\stdint.h(219): warning C
4005: 'WINT_MAX': macro redefinition
C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\INCLUDE\stdint.h(115): no
te: see previous definition of 'WINT_MAX'
c:\users\tvincent\desktop\fabio\fabio\ext\include\msvc\\stdint.h(230): warning C
4005: 'INT8_C': macro redefinition
C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\INCLUDE\stdint.h(117): no
te: see previous definition of 'INT8_C'
c:\users\tvincent\desktop\fabio\fabio\ext\include\msvc\\stdint.h(231): warning C
4005: 'INT16_C': macro redefinition
C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\INCLUDE\stdint.h(118): no
te: see previous definition of 'INT16_C'
c:\users\tvincent\desktop\fabio\fabio\ext\include\msvc\\stdint.h(232): warning C
4005: 'INT32_C': macro redefinition
C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\INCLUDE\stdint.h(119): no
te: see previous definition of 'INT32_C'
c:\users\tvincent\desktop\fabio\fabio\ext\include\msvc\\stdint.h(233): warning C
4005: 'INT64_C': macro redefinition
C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\INCLUDE\stdint.h(120): no
te: see previous definition of 'INT64_C'
c:\users\tvincent\desktop\fabio\fabio\ext\include\msvc\\stdint.h(235): warning C
4005: 'UINT8_C': macro redefinition
C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\INCLUDE\stdint.h(122): no
te: see previous definition of 'UINT8_C'
c:\users\tvincent\desktop\fabio\fabio\ext\include\msvc\\stdint.h(236): warning C
4005: 'UINT16_C': macro redefinition
C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\INCLUDE\stdint.h(123): no
te: see previous definition of 'UINT16_C'
c:\users\tvincent\desktop\fabio\fabio\ext\include\msvc\\stdint.h(237): warning C
4005: 'UINT32_C': macro redefinition
C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\INCLUDE\stdint.h(124): no
te: see previous definition of 'UINT32_C'
c:\users\tvincent\desktop\fabio\fabio\ext\include\msvc\\stdint.h(238): warning C
4005: 'UINT64_C': macro redefinition
C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\INCLUDE\stdint.h(125): no
te: see previous definition of 'UINT64_C'
c:\users\tvincent\desktop\fabio\fabio\ext\include\msvc\\stdint.h(241): warning C
4005: 'INTMAX_C': macro redefinition
C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\INCLUDE\stdint.h(127): no
te: see previous definition of 'INTMAX_C'
c:\users\tvincent\desktop\fabio\fabio\ext\include\msvc\\stdint.h(242): warning C
4005: 'UINTMAX_C': macro redefinition
C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\INCLUDE\stdint.h(128): no
te: see previous definition of 'UINTMAX_C'
fabio\ext\mar345_IO.c(3666): warning C4244: '=': conversion from 'Py_ssize_t' to
 'int', possible loss of data
fabio\ext\mar345_IO.c(3834): warning C4244: '=': conversion from 'Py_ssize_t' to
 'int', possible loss of data
fabio\ext\mar345_IO.c(3847): warning C4244: '=': conversion from 'Py_ssize_t' to
 'int', possible loss of data
fabio\ext\mar345_IO.c(4564): warning C4244: '+=': conversion from 'Py_ssize_t' t
o 'int', possible loss of data
fabio\ext\mar345_IO.c(4577): warning C4244: '+=': conversion from 'Py_ssize_t' t
o 'int', possible loss of data
fabio\ext\mar345_IO.c(4608): warning C4244: '+=': conversion from 'Py_ssize_t' t
o 'int', possible loss of data
fabio\ext\mar345_IO.c(4609): warning C4244: '+=': conversion from 'Py_ssize_t' t
o 'int', possible loss of data
fabio\ext\mar345_IO.c(4862): warning C4018: '<': signed/unsigned mismatch
fabio\ext\mar345_IO.c(5240): warning C4244: '=': conversion from '__pyx_t_5numpy
_int32_t' to '__pyx_t_5numpy_int16_t', possible loss of data
fabio\ext\mar345_IO.c(5260): warning C4018: '<': signed/unsigned mismatch
fabio\ext\mar345_IO.c(5294): warning C4244: '=': conversion from '__pyx_t_5numpy
_uint32_t' to '__pyx_t_5numpy_int16_t', possible loss of data
fabio\ext\mar345_IO.c(5304): warning C4244: '=': conversion from '__pyx_t_5numpy
_uint32_t' to '__pyx_t_5numpy_int16_t', possible loss of data
fabio\ext\mar345_IO.c(5314): warning C4244: '=': conversion from '__pyx_t_5numpy
_uint32_t' to '__pyx_t_5numpy_int16_t', possible loss of data
fabio\ext\mar345_IO.c(5335): warning C4244: '=': conversion from '__pyx_t_5numpy
_int32_t' to '__pyx_t_5numpy_int16_t', possible loss of data
fabio\ext\mar345_IO.c(5382): warning C4244: '=': conversion from '__pyx_t_5numpy
_uint32_t' to '__pyx_t_5numpy_int16_t', possible loss of data
fabio\ext\mar345_IO.c(7453): warning C4244: '=': conversion from '__pyx_t_5numpy
_int64_t' to '__pyx_t_5numpy_uint32_t', possible loss of data
fabio\ext\mar345_IO.c(9055): warning C4244: 'function': conversion from '__pyx_t
_5numpy_uint32_t' to '__pyx_t_5numpy_uint8_t', possible loss of data
fabio\ext\mar345_IO.c(9173): warning C4334: '<<': result of 32-bit shift implici
tly converted to 64 bits (was 64-bit shift intended?)
fabio\ext\mar345_IO.c(9656): warning C4244: '=': conversion from '__pyx_t_5numpy
_uint32_t' to '__pyx_t_5numpy_uint8_t', possible loss of data
fabio\ext\mar345_IO.c(9888): warning C4018: '>': signed/unsigned mismatch
fabio\ext\mar345_IO.c(11085): warning C4018: '<': signed/unsigned mismatch
fabio\ext\mar345_IO.c(11138): warning C4244: '=': conversion from '__pyx_t_5nump
y_int64_t' to '__pyx_t_5numpy_int32_t', possible loss of data
error: Command "C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\BIN\x86_a
md64\cl.exe /c /nologo /Ox /W3 /GL /DNDEBUG /MD -Ifabio\ext\include -Ifabio\ext
-IC:\Python36\lib\site-packages\numpy\core\include -IC:\Python36\include -IC:\Py
thon36\include -I"C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\INCLUDE
" -I"C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\ATLMFC\INCLUDE" -I"C
:\Program Files (x86)\Windows Kits\10\include\10.0.10586.0\ucrt" -I"C:\Program F
iles (x86)\Windows Kits\NETFXSDK\4.6.1\include\um" -I"C:\Program Files (x86)\Win
dows Kits\10\include\10.0.10586.0\shared" -I"C:\Program Files (x86)\Windows Kits
\10\include\10.0.10586.0\um" -I"C:\Program Files (x86)\Windows Kits\10\include\1
0.0.10586.0\winrt" /Tcfabio\ext\mar345_IO.c /Fobuild\temp.win-amd64-3.6\Release\
fabio\ext\mar345_IO.obj" failed with exit status 2
```

--- a/fabio/ext/include/ccp4_pack.h
+++ b/fabio/ext/include/ccp4_pack.h
@@ -19,7 +19,7 @@
 #ifndef CPP4_PACK_H
 #define CPP4_PACK_H
 
-#ifndef _MSC_VER
+#if !defined(_MSC_VER) || (_MSC_VER >= 1900)
 #include <stdint.h>
 #else
 #include "msvc\\stdint.h"

